app/views/admin/banners/_form.html.erb

```diff
@@ -34,12 +34,14 @@
   </div>
 
   <div class="row">
-    <div class="small-12 column">
-      <%= f.label :sections, t("admin.banners.banner.sections_label") %>
+    <fieldset class="small-12 column">
+      <legend><%= t("admin.banners.banner.legend") %></legend>
```

Comment:
Since we might eventually use other legends in the form, IMHO using `sections_legend` (or other word starting with `sections_`) makes it less likely to have conflicts in the future :thinking:.

Leaving it as `sections_label` is also fine with me because crowdin doesn't automatically rename keys for all languages and so translators will have to translate this text again if we rename the key.

Both solutions are fine with me, so choose the one you prefer :wink:.

Reply:
>Since we might eventually use other legends in the form, IMHO using sections_legend (or other word starting with sections_) makes it less likely to have conflicts in the future 🤔.

Nice catch 👍🏼

>Leaving it as sections_label is also fine with me because crowdin doesn't automatically rename keys for all languages and so translators will have to translate this text again if we rename the key.

I think Crowdin's translation memory would take care of this automaticallly but always better to reduce the changes so finally i will keep the previous translation key to not to bother ourselves with more Crowdin's commits.

Thanks @javierm